app/models/api/study_io.rb

```diff
@@ -99,5 +99,6 @@ def render_class
     map_attribute_to_json_attribute(:hmdmc_approval_number, 'hmdmc_number')
     map_attribute_to_json_attribute(:s3_email_list)
     map_attribute_to_json_attribute(:data_deletion_period)
+    map_attribute_to_json_attribute(:contaminated_human_data_access_group)
```

Comment:
Could you add an assertion for this in `spec/models/api/study_io_spec.rb` ?

Reply:
I am trying to add some code, Codecov is already complaining about it. Spoiler alert: I am totally new to the Ruby world, so it might take more time than expected.